### PR TITLE
Feature/tmpi 115/issue with select component

### PR DESCRIPTION
**Pull request by dorokhovich-mikhail**
Change p[rops in the select and fix styles for the Panel.

Please check the styles of the Panel component.

### CODE REVIEW

**Comment by johnny-five** [comment]
I, @tdakhla, have checked:

### Style/Standards/Process

* [ ] If there is commented out code that should be deleted
* [ ] That the change implements what was requested in the [ticket]()
* [ ] For adequate test coverage for changed lines or critical code paths

### Code Quality

* [ ] Do the classes follow [CodeClimate 7 Patterns to Refactor](http://blog.codeclimate.com/blog/2012/10/17/7-ways-to-decompose-fat-activerecord-models/)
* [ ] If there are more idiomatic ways of using our frameworks/libraries
* [ ] Are there alternative implementations or refactors that increase readability/understandability and/or maintainability
* [ ] I can actually understand what the code is doing (if you're struggling to understand, then others may as well)
* [ ] This change is ready for production, if not should there should be a [feature flag](https://github.com/voormedia/flipflop)

### Security and Privacy

* [ ] Do the changes raise any security or privacy concerns

File: src/Select/Select.component.tsx [hunk 1]
```diff
@@ -93,39 +93,55 @@ export class CustomSelect extends React.Component<Props> {
...
-            {...props}
             className="react-select-component"
-            isDisabled={this.props.disabled}
-            value={this.props.selectedOption}
+            isDisabled={disabled}
+            value={selectedOption}
             options={options}
-            {...props.invalid}
-            aria-invalid={this.props.invalid ? true : undefined}
+            invali={invalid}
```
Comment (tdakhla) [inline src/Select/Select.component.tsx]: This should be `invalid={invalid}`

Comment (dorokhovich-mikhail) [reply src/Select/Select.component.tsx]: @tdakhla fixied

**Comment by tdakhla** [comment]
LGTM

**Pull request merged**